### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by beechtom**
Part of puppetlabs/bolt#1707

### CODE REVIEW

**Comment by beechtom** [comment]
Something got messed up with the git history and the previous PR for this was auto-closed: https://github.com/puppetlabs/puppetlabs-gcloud_inventory/pull/3

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  for the service account
+
+## Usage
+
+The plugin generates targets from a list of compute engine instances for a specific project
+and zone. It supports the following fields:
+
+| Option | Type | Description |
+| ------ | ---- | ----------- |
+| `credentials` | `String` | An absolute path to the service account credentials file. _Optional._ |
```
Comment (nicklewis) [inline README.md]: Should this be absolute or should it be relative to the Boltdir (which would still allow absolute)?

Comment (beechtom) [reply README.md]: Do we have access to the Boltdir path from a task?

Comment (nicklewis) [reply README.md]: We do from plugins. It's passed as the `_boltdir` param.

https://github.com/puppetlabs/puppetlabs-terraform/blob/master/tasks/resolve_reference.rb#L48

Comment (beechtom) [reply README.md]: Ah, nice. I think it makes sense to have it be relative in that case.

**Review by nicklewis — APPROVED** [review]

**Comment by nicklewis** [comment]
I don't seem to have write access to this repo to merge this PR...

**Comment by beechtom** [comment]
@nicklewis Added @puppetlabs/bolt as admin.

**Pull request merged**